packages/html1-check/src/index.ts

```diff
@@ -0,0 +1,29 @@
+
+import { language } from '@html1/language-server/out/language';
+import { createHtml1Service } from '@html1/language-server/out/service';
+
+import createHtmlService from 'volar-service-html';
+import createCssService from 'volar-service-css';
+import * as kit from '@volar/kit';
+import * as path from 'path';
+
+const fileName = path.resolve(process.cwd(), process.argv[2]);
+
+const project = kit.createInferredProject(process.cwd(), () => [fileName]);
+const config: kit.Config = {
+	languages: {
+		html1: language,
+	},
+	services: {
+		html: createHtmlService(),
+		css: createCssService(),
+		html1: createHtml1Service(),
+	},
+};
```

Comment:
You can share large parts of the code with language-servers. This is the power of Volar.js abstraction :)